kubernetes/cluster.go

```diff
@@ -34,13 +34,13 @@ import (
 )
 
 const (
-	// APISGroupName is the api name used for carrier
-	APISGroupName = "carrier.suse.org"
+	// APISGroupName is the api name used for fuseml
+	APISGroupName = "fuseml.suse.org"
```

Comment:
how about "fuse.ml" instead of "fuseml.suse.org" ?

Reply:
why not